### DISCUSSION TITLE
Transparent Chatbox for Fixed - Classic Layout

### DISCUSSION THREAD

**Discussion by PureIllenium** [runelite/runelite#18012] [Ideas]
Is it possible for a plugin to be created that allows for a transparent chat box for the fixed - classic layout?

**Comment by PureIllenium** [comment]
@deathbeam could this be done within the fixed mode hide chat plugin?

**Reply by Nightfirecat** [reply]
If there were any plugin where this would fit, that would probably be it. That said, I don't think deathbeam plays osrs much at the moment, so it would likely need to be done by another contributor. (ideally adding it to that plugin)

**Reply by PureIllenium** [reply]
Yeah I figured deathbeam would be the most likely to put it in place either as new plugin or update to that one. I didn't realize deathbeam didn't play much osrs anymore so hopefully someone else looks to take it on

**Comment by TV-cmd** [comment]
Really good idea
